### PR TITLE
Resolve component governance bugs

### PR DESCRIPTION
**Pull request by wtgodbe**
Should backport this to 6.0.

### CODE REVIEW

File: src/content/React-CSharp/ClientApp/package.json [hunk 1]
```diff
@@ -67,5 +67,13 @@
...
       "last 1 safari version"
     ]
+  },
+  "resolutions": {
+    "ejs": ">=3.1.6",
+    "immer": ">=9.0.6",
+    "url-parse": ">=1.5.0",
+    "lodash": ">=4.17.21",
+    "glob-parent": ">=5.1.2",
+    "set-value": "4.0.1"
```
Comment (javiercn) [inline src/content/React-CSharp/ClientApp/package.json:77]: Our templates use NPM, these don't do anything

Comment (wtgodbe) [reply src/content/React-CSharp/ClientApp/package.json]: Updated

Comment (javiercn) [reply src/content/React-CSharp/ClientApp/package.json]: I apologize will, I'm in the middle of doing a dep update (the traditional way) myself, just want to make sure I don't waste your time.

**Comment by wtgodbe** [comment]
@javiercn @HaoK @pranavkm any objections to this?

**Comment by javiercn** [comment]
@wtgodbe https://github.com/dotnet/spa-templates/pull/31

**Comment by wtgodbe** [comment]
This is now a direct port of https://github.com/dotnet/spa-templates/pull/31 (CC @pranavkm @mkArtakMSFT @javiercn)

**Pull request merged**